packages/manager/modules/octavia-load-balancer/src/octavia-load-balancer.routing.js

```diff
@@ -1,53 +1,13 @@
-import template from './octavia-load-balancer.html';
-
 export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('octavia-load-balancer', {
     url: '/pci/projects/{projectId:[0-9a-zA-Z]{32}}/octavia-load-balancer',
-    template,
-    redirectTo: 'octavia-load-balancer.onboarding',
+    redirectTo: () => 'octavia-load-balancer.loadbalancers',
     resolve: {
       projectId: /* @ngInject */ ($transition$) =>
         $transition$.params().projectId,
       project: /* @ngInject */ ($http, projectId) =>
         $http.get(`/cloud/project/${projectId}`).then(({ data }) => data),
-      breadcrumb: ($translate) => $translate.instant('octavia_load_balancer'),
```

Comment:
I think breadcrumb and breadcrumb prefix should stay on parent state as it is used by other states than octavia-load-balancer.loadbalancers

Reply:
Done